src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -159,12 +161,13 @@ public static function getOutbox($owner, $page = null)
 
 		$data = ['@context' => ActivityPub::CONTEXT];
 		$data['id'] = System::baseUrl() . '/outbox/' . $owner['nickname'];
-		$data['type'] = 'OrderedCollection';
```

Comment:
Can you please decide on one consistent way of enabling your change, either by moving all the types declarations to the  `if (empty($page))` conditions or leaving all default type declaration out of them?

Reply:
I was to move to the `empty($page)` conditions. 0c732f2

Reply:
Sorry, I forgot the `! empty ($ profile ['hide-friends'])` conditions. I will to use the latter method.